pennylane/interfaces/batch/unwrap.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains a context manager for unwrapping tapes
+"""
+import pennylane as qml
+
+
+class UnwrapTape:
+    """A context manager that unwraps a tape with TensorFlow parameters

```

Comment:
Is the "TensorFlow" here a typo? (I guess so!)

Reply:
Yep, oops